chunky/src/java/se/llbit/chunky/ui/render/tabs/AdvancedTab.java

```diff
@@ -190,6 +192,24 @@ public PictureExportFormat fromString(String string) {
             }));
     bvhMethod.setTooltip(new Tooltip(bvhMethodBuilder.toString()));
 
+    ArrayList<String> biomeStructureNames = new ArrayList<>();
+    StringBuilder biomeStructureTooltipBuilder = new StringBuilder();
+    for(Map.Entry<String, BiomeStructure.Factory> entry : BiomeStructure.REGISTRY.entrySet()) {
+      biomeStructureNames.add(entry.getKey());
+      biomeStructureTooltipBuilder.append(entry.getKey());
+      biomeStructureTooltipBuilder.append(": ");
+      biomeStructureTooltipBuilder.append("Description"); //entry.getValue().getDescription());
+      biomeStructureTooltipBuilder.append('\n');
+    }
+    biomeStructureTooltipBuilder.append("Requires reloading chunks to take effect.");
+    biomeStructureImplementation.getItems().addAll(biomeStructureNames);
+    biomeStructureImplementation.getSelectionModel().selectedItemProperty()
+      .addListener((observable, oldvalue, newvalue) -> {
+        scene.setBiomeStructureImplementation(newvalue);
```

Comment:
A prompt (as for the emitter grid implementation) would be nice in a follow-up PR :+1: